### PR TITLE
Adding MixedLoadTest to otherLoadTest as part of Load_Level_4 playlist

### PR DESCRIPTION
**Pull request by psoujany**
I've added a new test called MixedLoadTest to otherLoadTest as part of Load_Level_4 playlist which is a mix of workloads.

### CODE REVIEW

**Comment by psoujany** [comment]
Successful test run
https://hyc-runtimes-team-svt-jenkins.swg-devops.com/view/Test_grinder/job/Grinder/268/console

File: system/otherLoadTest/playlist.xml [hunk 1]
```diff
@@ -300,4 +300,20 @@
...
+		</variations>
+		<command>$(SYSTEMTEST_CMD_TEMPLATE) -test=MixedLoadTest; \
+	$(TEST_STATUS)</command>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>system</group>
+		</groups>
+	</test>
```
Comment (Mesbah-Alam) [inline system/otherLoadTest/playlist.xml]: Please add: 

```
<impls>
	<impl>ibm</impl>
</impls>
```

Here. As we only want this test to run when we are testing IBM SDKs.

Comment (psoujany) [reply system/otherLoadTest/playlist.xml]: Ok, I'll add it.

**Comment by psoujany** [comment]
@Mesbah-Alam 
I've made above changes. Thank you.

**Pull request merged**